src/mesh/structured_mesh.jl

```diff
@@ -0,0 +1,15 @@
+struct StructuredMesh{RealT<:Real, NDIMS}
+  size::Tuple{Vararg{Int64, NDIMS}}
+  coordinates_min::Tuple{Vararg{RealT, NDIMS}}
+  coordinates_max::Tuple{Vararg{RealT, NDIMS}}
+  linear_indices::LinearIndices{NDIMS, Tuple{Vararg{UnitRange{Int64}, NDIMS}}}
```

Comment:
```suggestion
  size::NTuple{NDIMS, Int}
  coordinates_min::NTuple{NDIMS, RealT}
  coordinates_max::NTuple{NDIMS, RealT}
  linear_indices::LinearIndices{NDIMS, NTuple{NDIMS, UnitRange{Int}}}
```
Is there a reason to use specifically `Int64`or the more verbose (and for me less readable) version of `NTuple` that I didn't see?

Reply:
No, there is not. We did not know about `NTuple` and `Int`.